tests/QMLEngine/qml/PropertiesAndSignalsWithTheSameNames2.qml

```diff
@@ -0,0 +1,20 @@
+import QtQuick 2.5
+Item {
+  property bool signalCaught: false
+  property Item area: mouse_area
+  signal pressed()
+  property bool pressed: false
+  MouseArea
+  {
```

Comment:
Please use consistent formatting if possible. E.g that would be `MouseArea {` on lines 7-8.